docs/recipes/quickly-switch between-multiple-foam-repos.md

```diff
@@ -0,0 +1,32 @@
+# Quickly switch between multiple Foam repos
+
+this #recipe is allow to Quickly switch between multiple Foam repos. and sorry for Setting/Content's Hangul the Korean Character.
```

Comment:
Text is not Capitalized correctly:  "this" -> "This".  "and" -> "And".

Also not entirely sure what the sorry is about.. Could you just fix the issue instead?